recipes/quartet/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "quartet" %}
+{% set version = "1.0" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/ngodber/quartet/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: eaeebc506d743d59f04bb24725df5ed01d26775f8d94c73a70ab8838151a9303
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+  host:
+  run:
+
+test:
+  imports:
+  requires:
+  commands:
+    - test -f ${PREFIX}/bin/quartet  # [unix]
+    - test -f ${PREFIX}/lib/libquartet.a  # [unix]
+    - if not exist %LIBRARY_PREFIX%\bin\quartet.exe exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\lib\quartet.lib exit 1  # [win]
+
+about:
+  home: https://github.com/PyMesh/quartet
+  license: Custom
```

Comment:
Please use the correct SPDX specifier for this, should be `ISC` from a quick look.